src/ansys/fluent/core/quantity.py

```diff
@@ -708,7 +708,7 @@ def __eq__(self, other):
         if isinstance(other, Quantity):
```

Comment:
Maybe you can use `validate_matching_dimensions` here too

Reply:
Right. Done. Thank you.

Reply:
Right. Done. Thank you.